### PR TITLE
feat: new send tests

### PR DESCRIPTION
**Pull request by limpbrains**
- new send wallet-send test
- rbf now false by default in `setupOnChainTransaction`
- `setupOnChainTransaction`, `updateBitcoinTransaction`, `updateSelectedFeeId` and `resetOnChainTransaction` can now return errors
- fix using of `InteractionManager.runAfterInteractions` in `resetLdk`
- replace `new Buffer` with `Buffer.from` to remove deprication warning

### CODE REVIEW

File: src/store/actions/wallet.ts [hunk 1]
```diff
@@ -821,7 +821,7 @@ export const setupOnChainTransaction = async ({
 	selectedNetwork,
 	addressType,
 	inputTxHashes,
-	rbf = true,
+	rbf = false,
```
Comment (pwltr) [inline src/store/actions/wallet.ts:824]: Do we need to pass `rbf: true` now for usage of `setupOnChainTransaction` in `setupRbf` function?

File: __tests__/wallet-send.ts [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+	updateFee,
+	validateTransaction,
+} from '../src/utils/wallet/transactions';
+import { addElectrumPeer } from '../src/store/actions/settings';
+import { getScriptHash } from '../src/utils/wallet';
+import initWaitForElectrumToSync from './utils/wait-for-electrum';
+
+jest.setTimeout(60_000);
+
+const bitcoinURL =
```
Comment (pwltr) [inline __tests__/wallet-send.ts:27]: Do I need to change this URL to be able to run the tests locally? I have the containers running and ran chmod on the `.cookie` file.

File: __tests__/wallet-send.ts [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+	'http://electrumx:1VmSUVGBuLNWvZl0LExRDW0tvl6196-47RfXIzS384g=@localhost:43782';
```
Comment (pwltr) [inline __tests__/wallet-send.ts:28]: Do I need to change this URL to be able to run the tests locally? I have the containers running and ran chmod on the .cookie file.

File: src/store/actions/wallet.ts (hunk 1 above)
Comment (limpbrains) [reply src/store/actions/wallet.ts:824]: Nice catch! I should probably revert it or add more tests, thx

File: __tests__/wallet-send.ts (hunk 3 above)
Comment (limpbrains) [reply __tests__/wallet-send.ts:28]: You don't if you are running `docker-compose up` from `__tests__`. 
chmod needed on linux, on mac you don't have to do this

Comment (pwltr) [reply __tests__/wallet-send.ts:28]: Working now!

File: src/store/actions/wallet.ts (hunk 1 above)
Comment (limpbrains) [reply src/store/actions/wallet.ts:824]: I've fixed usage of setupOnChainTransaction in setupRbf, please review again

**Review by pwltr — APPROVED** [review]

**Pull request merged**